### DISCUSSION TITLE
Expansion Error: BUG: The expected size 2147483648 of engine pvc-25bd7bb8-3c72-43d6-8bd9-17998eb76fce-e-f81fcd0b should not be smaller than the current size 26843545600 Note: You can cancel the expansion to avoid volume crash

### DISCUSSION THREAD

**Discussion by ankitkaushik94** [longhorn/longhorn#6698] [General]
Hi Team,

We are using Longhorn as a Storage for our K8s cluster.
We've upgraded to v1.4.2 recently. After that some of the volumes are going into infinite expansion mode.
I've tried to cancel the expansion but it throws an error saying "unable to cancel expansion for volume pvc-25bd7bb8-3c72-43d6-8bd9-17998eb76fce: volume expansion is not started".
It is not coming out of this state from a very long time. Same is happening with other volumes as well.
Could you please suggest what can be done in this case as due to this our pods are not running.

Best,
Ankit Kaushik

**Comment by mayuresh-quartic** [comment]
I have also oberserved same in Longhorn version 1.4.0
Cancel expansion is also not working and it throws error Expansion process has not started yet

**Reply by ankitkaushik94** [reply]
Were you able to fix this issue?

**Reply by mayuresh-quartic** [reply]
Nope
